### PR TITLE
Feat/support mimo

### PR DESCRIPTION
**Pull request by pengchengneo**
## Motivation

support XiaoMi-mimo model 

## Accuracy Tests
launcher server
```
 uv run python3 -u -m sgl_jax.launch_server   --model-path XiaomiMiMo/MiMo-7B-RL   --trust-remote-code   --device=tpu    --mem-fraction-static=0.8   --max-prefill-tokens=4096   --max-running-requests=128  --attention-backend=fa   --dtype=bfloat16  --port 30000 --host 0.0.0.0 --tp-size 4 --page-size 64
```
run eval test
```
uv run evalscope eval   --model XiaomiMiMo/MiMo-7B-RL   --api-url http://127.0.0.1:30000/v1/chat/completions   --datasets math_500   --eval-type service   --generation-config do_sample=true,temperature=0.6   --generation-config bos_token_id=151643,do_sample=false,eos_token_id=151643,max_new_tokens=2048,transformers_version=4.37.0 --eval-batch-size 128
```

launcher server with mtp
```
uv run python3 -u -m sgl_jax.launch_server   --model-path XiaomiMiMo/MiMo-7B-RL   --trust-remote-code   --device=tpu    --mem-fraction-static=0.5   --max-prefill-tokens=4096   --max-running-requests=16  --decode-log-interval=1   --enable-request-time-stats-logging   --attention-backend=fa   --dtype=bfloat16  --port 30000 --host 0.0.0.0 --disable-overlap-schedule --tp-size 4 --speculative-algorithm EAGLE --speculative-draft-model-path  XiaomiMiMo/MiMo-7B-RL --page-size 64 --speculative-eagle-topk 1 --speculative-num-steps 3 --speculative-num-draft-tokens 4 
```
image

### CODE REVIEW

File: python/sgl_jax/srt/kernels/ragged_paged_attention/tuned_block_sizes.py [hunk 1]
```diff
@@ -1242,10 +1245,18 @@ def get_tuned_block_sizes(
...
+                device_name,
+                page_size,
+                actual_num_q_heads,
+                actual_num_kv_heads,
+                head_dim,
+                max_num_tokens,
+                pages_per_seq,
             )
-            print(f"Using default block size: {bkv_p=}, {bq=}.")
+            logger.debug("Using default block size: bkv_p=%s, bq=%s.", bkv_p, bq)
```
Comment (JamesBrianD) [inline python/sgl_jax/srt/kernels/ragged_paged_attention/tuned_block_sizes.py]: I think that log level could be log, because it is only printed once in tracing

Comment (pengchengneo) [reply python/sgl_jax/srt/kernels/ragged_paged_attention/tuned_block_sizes.py]: ok
